include/pfs/procfs.hpp

```diff
@@ -73,6 +73,8 @@ class procfs final
 
     std::string get_version_signature() const;
 
+    std::vector<route> get_routes() const;
```

Comment:
Since the file is located under `/proc/net/route`, this should move under `net.hpp` and be called `get_route`

Reply:
The convention is to use `get_<filename>` where filename is the exact file under `/proc/net/`.
Can we use `get_route` instead of `get_routes` for consistency?